packages/contracts/src/abis/Unirep.ts

```diff
@@ -0,0 +1,1595 @@
+export const UnirepABI = [
```

Comment:
I also think we should gitignore contract ABIs. We can auto-generate this file when we generate typechain files instead.